### DISCUSSION TITLE
DeltaP in Ketton tutorial

### DISCUSSION THREAD

**Discussion by foojan-Ka** [GeoChemFoam/GeoChemFoam#18] [Q&A] [answered]
Hello,

When I was reading through the Test Case 01 (Ketton), the pressure difference which is DeltaP is defined in Pascal by the user in initCaseFlow.sh.
However, in the "p" file of "0_org" folder, the defined pressure is kinematic pressure (m2/s2). I just wanted to ask shouldn't we define the input pressure (deltaP) in kinematic pressure instead of Pascal value?

Looking forward to hearing from you.

Best,
Foojan

**Comment by julienmaes** [comment, marked as answer]
Yes that's a typo. It's in m2/s2

**Reply by foojan-Ka** [reply]
Thanks for your reply.
So, I have to input pressure difference over density in deltaP value?
And the pressure results in Paraview should be multiplied by the density also to obtain the pressure in Pascal in the end?

**Comment by julienmaes** [comment]
Yes. Note that the viscosity is the kinematic viscosity, you also need to multiply it by density to get the dynamic viscosity. So you don't have to multiply anything if you're calculating the permeability
